xml/deployment_prep_zseries_prep_i.xml

```diff
@@ -732,9 +685,9 @@ MDISK 151 3390 0253 2800 DASD40 MR ONE4ME TWO4ME THR4ME</screen>
      the Linux installation.
     </para>
     <para>
-     Now add (as the user
+     Now add as the user
```

Comment:
I would avoid the parenthesis altogether and start the sentence with: 'As user ..., add the guest to'